docker/production.yml

```diff
@@ -31,6 +31,7 @@ services:
         - POSTS_URL=${POSTS_URL}
         - SEARCH_URL=${SEARCH_URL}
         - FEED_DISCOVERY_URL=${FEED_DISCOVERY_URL}
+        - DEPENDENCY_DISCOVERY_URL=${DEPENDENCY_DISCOVERY_URL}
```

Comment:
And here.

Also, is there a reason we don't define these values once in `dockr-compose.yml` and then not in all the other ones?  Are they different?  If not, we just need it once in the main file, and these overrides can not touch `environment`.

Reply:
Actually, those args are not needed here, since we don't run our Next.js in this service.
It's under nginx.